src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp

```diff
@@ -300,15 +336,15 @@ APPLICATION_INFO::FindRequestHandlerAssembly(STRU& location)
 
             if (g_hAspnetCoreRH == NULL)
             {
-                FINISHED(HRESULT_FROM_WIN32(GetLastError()));
+                FINISHED_IF_FAILED(HRESULT_FROM_WIN32(GetLastError()));
             }
         }
 
         g_pfnAspNetCoreCreateApplication = (PFN_ASPNETCORE_CREATE_APPLICATION)
             GetProcAddress(g_hAspnetCoreRH, "CreateApplication");
         if (g_pfnAspNetCoreCreateApplication == NULL)
         {
-            FINISHED(HRESULT_FROM_WIN32(GetLastError()));
+            FINISHED_IF_FAILED(HRESULT_FROM_WIN32(GetLastError()));
```

Comment:
Why change this?

Reply:
My bad on the merge conflicts. Revert this.